Classes/Notifications/NoNewNotificationsCell.swift

```diff
@@ -51,6 +53,9 @@ final class NoNewNotificationsCell: UICollectionViewCell {
                 name: NSNotification.Name.UIApplicationWillEnterForeground,
                 object: nil
         )
+        
+        contentView.isAccessibilityElement = true
+        contentView.accessibilityLabel = NSLocalizedString("You have no new notifications!", comment: "Inbox Zero Accessibility Label")
```

Comment:
Omg, congrats on the first `NSLocalizedString` comment 😂

Reply:
😂 I've moved onto an international team at work so I've begun having localization thrown down my throat - Still working on it though